amr-wind/ocean_waves/relaxation_zones/waves2amr_ops.H

```diff
@@ -339,59 +339,51 @@ struct ReadInputsOp<W2AWaves>
                 std::to_string(flag));
         }
 
-        // If init_wave_field is activated and initialization will be done, get
-        // modes on every processor
-        if (wdata.init_wave_field && data.sim().time().time_index() == 0) {
```

Comment:
this looks like a lot of changes here, but really the only change within ReadInputsOp is removing this conditional. The first part needed to be remove to fix a segfault under certain conditions, and the second part did nothing because time_index always == 0 during the read inputs phase of the algorithm because the current time hasn't been calculated yet.